Examples/GetFullPredictions/Program.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using IndicoV2;
+using Newtonsoft.Json.Linq;
+
+namespace Examples
+{
+    /// <summary>
+    /// Returns full output with pre_review, post_review, and final predictions
+    /// </summary>
+    internal class GetFullPredictions
+    {
+        private static string GetToken() =>
+            File.ReadAllText(Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile),
+                "indico_api_token.txt"));
+
+        public static async Task Main()
+        {
+            var client = new IndicoClient(GetToken(), new Uri("https://app.indico.io"));
+
+            var dataSets = await client.DataSets().ListAsync();
+
+            var workflows = await client.Workflows().ListAsync(dataSets.First().Id);
```

Comment:
I'm personally of the opinion that you write a workflowID into the example vs. include some methods to get a workflow ID that no one would ever actually use-- but curious what Meg thinks (obviously makes testing them impossible)

Reply:
I believe we typically either show how to get a workflow id from a dataset id or just hardcode it in the python examples -- so following that here would be good. People can hand change the workflow id or dataset id if they need to. 

adding comments explaining what is happening or pointing to where to find workflow id can help, too.